src/Microsoft.DotNet.Build.Tasks/PackageFiles/Build.Common.targets

```diff
@@ -8,6 +8,9 @@
 
   <PropertyGroup>
     <BuildToolsTaskDir Condition="'$(BuildToolsTaskDir)' == ''">$(ToolsDir)</BuildToolsTaskDir>
+    
+    <!-- For UAP, we need to bin place the resources as resw files to create the runner's pri file. -->
+    <ResourcesFolderPath Condition="'$(ResourcesFolderPath)' == ''">$(RuntimePath)resw</ResourcesFolderPath>
```

Comment:
This looks like it will cause them to be bin-placed for all configurations. Is that correct?